packages/mesh-browser/go/jsutil/jsutil.go

```diff
@@ -7,10 +7,43 @@ package jsutil
 import (
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
+	"runtime/debug"
 	"syscall/js"
 )
 
+func CopyBytesToJS(bytes []byte) (jsBytes js.Value, err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			err = RecoverError(r)
+		}
+	}()
+
+	jsBytes = js.Global().Get("Uint8Array").New(len(bytes))
+	copied := js.CopyBytesToJS(jsBytes, bytes)
+	if copied != len(bytes) {
+		return js.Undefined(), fmt.Errorf("should have copied %d bytes to JS but only copied %d", len(bytes), copied)
+	}
+	return jsBytes, nil
+}
+
+func CopyBytesToGo(jsBytes js.Value) (bytes []byte, err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			err = RecoverError(r)
+		}
+	}()
+
+	jsBytesLength := jsBytes.Get("length").Int()
```

Comment:
Not worth it. Just want to make sure it fails immediately and doesn't cause confusing problems later on. Are you sure it only accepts `Uint8Array` and not any object that has a `length` field?

Reply:
According to the docs it will panic if `dst` is not of type `UintArray8`: https://golang.org/pkg/syscall/js/#CopyBytesToJS. The same is true of `CopyBytesToGo` (except that the type checking happens on `src`): https://golang.org/pkg/syscall/js/#CopyBytesToGo.